packages/graphiql/__mocks__/@graphiql/react.tsx

```diff
@@ -62,6 +62,7 @@ export {
   PenIcon,
   PlayIcon,
   PluginContext,
+  PluginContextProfider,
```

Comment:
That's what CI is complaining about
```suggestion
  PluginContextProvider,
```